backend/src/question-service/controllers/questions.ts

```diff
@@ -45,4 +46,31 @@ export const addQuestion: RequestHandler[] = [
   },
 ];
 
-// TODO: updateQuestion, deleteQuestion
+export const deleteQuestion: RequestHandler = async (req, res, next) => {
+  try {
+    const questionId = req.params.questionId;
+
+    // regex to check if questionId is a number
+    if (!/^\d+$/.test(questionId)) {
```

Comment:
possible alternative:
`param("questionId").isInt()` from express-validator

but this is fine too :P